.ci/functional_test.sh

```diff
@@ -55,7 +55,7 @@ echo $page | grep '<a class="button" href="/auth/identity">Sign in with email</a
 
 echo "Testing telemetry_id survives reconfigures"
 tid1=`sudo cat /var/opt/firezone/cache/telemetry_id`
-sudo firezone-ctl reconfigure
+sudo -E firezone-ctl reconfigure
```

Comment:
Just ensures `DISABLE_TELEMETRY` is there so that we don't through our metrics off with CI runs